### DISCUSSION TITLE
请问如何把api封装成可多线程使用呢

### DISCUSSION THREAD

**Discussion by funcxs** [lenML/Speech-AI-Forge#63] [问答 | Q&A]
请问如何把api封装成可多线程使用呢

**Comment by zhzLuke96** [comment]
可以看这个文档
https://github.com/lenML/ChatTTS-Forge/blob/main/docs/api.md#api

cli 参数 `--workers` 可指定并发数

相关的参数，还可以使用 `--device_id` 指定使用的 device id
